plugins/BEdita/API/src/Policy/ObjectPolicy.php

```diff
@@ -64,6 +66,42 @@ public function canUpdate(IdentityInterface $identity, ObjectEntity $object): bo
         return !empty(array_intersect($permsRoles, $this->extractRolesNames($identity)));
     }
 
+    /**
+     * Check if $identity can update parents of $object.
+     *
+     * @param \Authorization\IdentityInterface $identity The identity.
+     * @param \BEdita\Core\Model\Entity\ObjectEntity $object The object entity
+     * @return bool
+     */
+    public function canUpdateParents(IdentityInterface $identity, ObjectEntity $object): bool
+    {
+        /** @var \BEdita\Core\Model\Entity\ObjectType $folderObjectType */
+        $folderObjectType = $this->fetchTable('ObjectTypes')->get('folders');
+        if (!$folderObjectType->hasAssoc('Permissions')) {
+            return true;
+        }
+
+        if ($object instanceof Folder) {
+            return $this->canUpdate($identity, $object);
+        }
+
+        $parents = $this->fetchTable('Folders')
+            ->find('available')
+            ->contain(['Permissions.Roles'])
+            ->innerJoinWith('Children', function (Query $q) use ($object) {
+                return $q->where(['Children.id' => $object->id]);
+            })
```

Comment:
[minor] Anonymous functions should be supported since PHP 7.4. It would make this condition slightly more compact
```suggestion
            ->innerJoinWith('Children', fn (Query $q) => $q->where(['Children.id' => $object->id]))
```